test/e2e/app-dir/next-after-app/index.test.ts

```diff
@@ -26,25 +26,42 @@ describe.each(runtimes)('unstable_after() in %s runtime', (runtimeValue) => {
 
   if (skipped) return
 
+  const filesToPatchRuntime = [
+    'app/layout.js',
+    'app/route/route.js',
+    'app/route-streaming/route.js',
+  ]
+  const replaceRuntime = (contents: string, file: string) => {
+    const placeholder = `// export const runtime = 'REPLACE_ME'`
+
+    if (!contents.includes(placeholder)) {
+      throw new Error(`Placeholder "${placeholder}" not found in ${file}`)
+    }
+
+    return contents.replace(
+      placeholder,
+      `export const runtime = '${runtimeValue}'`
+    )
+  }
+
+  const runtimePatches = new Map<
+    string,
+    string | ((contents: string) => string)
+  >(
+    filesToPatchRuntime.map(
+      (file) =>
+        [file, (contents: string) => replaceRuntime(contents, file)] as const
```

Comment:
Rather than abstracting this here with custom replacement mechanics, why not just add another route/page, re-export the same components/handlers and add `export const runtime = "edge"`?

This extra test file manipulation is fragile.